data/bhattabhi013.json

```diff
@@ -0,0 +1,62 @@
+{
+    "name": "Abhishek Bhatt",
+    "displayStatsPublic": true,
+    "type": "personal",
+    "bio": "Full-Stack developer | Flutter | NodeJs | Angular",
+    "avatar": "https://github.com/bhattabhi013.png",
+    "tags": ["flutter", "nodejs", "angular", "javascript", "typescript", "open-source"],
+    "socials": [
+      { "platform": "twitter", "url": "https://twitter.com/AbhishekBtwt" },
+      { "platform": "github", "url": "https://github.com/bhattabhi013" },
+      { "platform": "instagram", "url": "https://www.instagram.com/bwith_abhi" }
```

Comment:
Is Instagram available in this field? 🤔

Reply:
I am not sure, let's deploy and find out 😂  if not we can add it